packages/components/src/toolbar/stories/index.tsx

```diff
@@ -74,7 +82,7 @@ Default.args = {
 			</ToolbarGroup>
 			<ToolbarGroup>
 				<ToolbarItem>
-					{ ( toggleProps ) => (
+					{ ( toggleProps: unknown ) => (
```

Comment:
Instead of `unknown`, should this also be a `@ts-expect-error` for when `ToolbarItem`/`DropdownMenu` is typed?

Reply:
Agreed! Covered in [dcc7a3a](https://github.com/WordPress/gutenberg/pull/47087/commits/dcc7a3ad9f7474b1efede6e69615ce68d7617e0a).